Cleanup

Zemurin opened the pull request:
- regions
- relations
- religions

Idhrendur commented on EU4ToVic2Tests/MapperTests/CultureMapperTests.cpp:
```diff
@@ -19,7 +19,7 @@ CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
 SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.*/
 
-
+/*
```
Is there a reason these are all commented out? If so, please create a todo tied to a github issue to deal with that reason. If not, please restore them.

Zemurin replied:
they rely on a mock I deleted when cleaning regions. mappers are next on my list so I'll be revising this in the next PR.

Idhrendur reviewed (approved):
It looks like the usual light changes. Anything I should have looked at more closely?

Zemurin commented:
No logic changes if that's what you're asking. I dropped entire religion block that was unused and cleaned up a lot of minor details, but it's mostly cosmetics and tests.
